services/order.py

```diff
@@ -0,0 +1,32 @@
+from django.db import transaction
+
+from db.models import Order, Ticket, User
+
+
+def create_order(tickets: list[dict], username: str, date: str = None) -> None:
+    with transaction.atomic():
+        order = Order.objects.create(
+            user=User.objects.get(username__exact=username)
+        )
+
+        if date:
+            order.created_at = date
+
+        order.save()
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                row=ticket["row"],
+                seat=ticket["seat"],
+                movie_session_id=ticket["movie_session"],
+                order=order
+            )
```

Comment:
use bulk_create for this action

Reply:
Because in this way you generate len(tickets) queries

Reply:
Using bulk_create() fails tests in this project